client/src/shared/LoginInputHandler.tsx

```diff
@@ -0,0 +1,27 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+export const passwordRegExp =
+  /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/;
+
+const signInSchema = Yup.object({
+  email: Yup.string().email("Invalid email").required("Email is required"),
+  password: Yup.string()
+    .required("Password is required")
+    .min(4, "Password is too short - should be 4 chars minimum")
```

Comment:
I think something here is wrong, firstly password must be minimum 4 characters and then 8 characters? :D